src/renderer/components/+helm-charts/helm-charts.tsx

```diff
@@ -37,6 +39,8 @@ interface Dependencies {
 
 @observer
 class NonInjectedHelmCharts extends Component<Dependencies> {
+  private readonly imgLoadingFailed = observable.set<string>(); // The IDs of the HelmChart instances
```

Comment:
We should use self-explanatory variable name, this will allow us to avoid such comments.